tvb/simulator/models/__init__.py

```diff
@@ -46,8 +46,10 @@
 from .oscillator import Generic2dOscillator, Kuramoto
 from .larter_breakspear import LarterBreakspear
 from .wong_wang import ReducedWongWang
+from .wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
 from .linear import Linear
 from .hopfield import Hopfield
-from .epileptor import Epileptor
+from .epileptor import Epileptor, Epileptor2D
+from .JCepileptor import JC_Epileptor
 from .epileptorcodim3 import EpileptorCodim3, EpileptorCodim3SlowMod
-from .Zerlaut import Zerlaut_adaptation_first_order, Zerlaut_adaptation_second_order
\ No newline at end of file
+from .Zerlaut import Zerlaut_adaptation_first_order, Zerlaut_adaptation_second_order
```

Comment:
I see a few extraneous imports here, should you rebase this against the latest trunk? if the Zerlaut models are not in trunk, the import lines fail.  If they are, you shouldn't need this change

Reply:
Zerlaut models are in trunk already.
The last line is not really changed with my current commit, only newline was added automatically by my IDE.
But maybe I do not understand completely your comment....

Reply:
I see what you mean.  it's not a problem, I just didn't expect to see extra diff lines that aren't for the model being added.

Reply:
True.... 
I also did a review of this init file, in addition to adding the current model, and did not reflect that in the commit comment.